### PR TITLE
Functional tests - Add test 'Send a test email'

### PR DESCRIPTION
**Pull request by boubkerbribri**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.7.x
| Description?  | Add test 'Send a test email'
| Type?         | refacto
| Category?     | TE
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | no
| How to test?  | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/18490)

### CODE REVIEW

File: tests/puppeteer/pages/BO/advancedParameters/email/index.js [hunk 1]
```diff
@@ -25,6 +26,12 @@ module.exports = class Email extends BOBasePage {
     this.selectAllRowsLabel = `${this.emailGridPanel} tr.column-filters .md-checkbox i`;
     this.bulkActionsToggleButton = `${this.emailGridPanel} button.js-bulk-actions-btn`;
     this.bulkActionsDeleteButton = '#email_logs_grid_bulk_action_delete_email_logs';
+
+    // Test your email configuration form
+    this.sendTestEmailForm = 'form[name=\'test_email_sending\']';
+    this.sendTestEmailInput = '#test_email_sending_send_email_to';
+    this.sendTestEmailButton = `${this.sendTestEmailForm} button.js-send-test-email-btn`;
+    this.sendTestEmailAlertParaph = `${this.sendTestEmailForm} .alert-success p.alert-text`;
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/advancedParameters/email/index.js]:
```suggestion
    this.sendTestEmailAlertParagraph = `${this.sendTestEmailForm} .alert-success p.alert-text`;
```

File: tests/puppeteer/pages/BO/advancedParameters/email/index.js [hunk 2]
```diff
@@ -142,4 +149,15 @@ module.exports = class Email extends BOBasePage {
     await this.clickAndWaitForNavigation(this.bulkActionsDeleteButton);
     return this.getTextContent(this.alertSuccessBlockParagraph);
   }
+
+  /**
+   * Send a test email
+   * @param email
+   * @return {Promise<string>}
+   */
+  async sendATestEmail(email) {
```
Comment (Progi1984) [inline tests/puppeteer/pages/BO/advancedParameters/email/index.js]:
```suggestion
  async sendTestEmail(email) {
```

Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/advancedParameters/email/index.js]: done thanks

File: tests/puppeteer/pages/BO/advancedParameters/email/index.js (hunk 1 above)
Comment (boubkerbribri) [reply tests/puppeteer/pages/BO/advancedParameters/email/index.js]: done thanks

**Review by Progi1984 — APPROVED** [review]

**Review by NeOMakinG — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Pull request merged**